docs/index.asciidoc

```diff
@@ -25,18 +25,16 @@ based on data from the MaxMind GeoLite2 databases.
 
 ==== Supported Databases
 
-This plugin is bundled with https://dev.maxmind.com/geoip/geoip2/geolite2[GeoLite2] City database out of the box. From MaxMind's description --
+This plugin is bundled with https://dev.maxmind.com/geoip/geoip2/geolite2[GeoLite2] ASN and City databases out of the box. From MaxMind's description --
 "GeoLite2 databases are free IP geolocation databases comparable to, but less accurate than, MaxMind’s
 GeoIP2 databases". Please see GeoIP Lite2 license for more details.
 
-https://www.maxmind.com/en/geoip2-databases[Commercial databases] from MaxMind are also supported in this plugin.
+https://www.maxmind.com/en/geoip2-databases[Commercial databases] Country, City, City Database by continent, ISP and Domain from MaxMind are also supported in this plugin.
```

Comment:
@karenzone We don't have commercial databases to test locally. All are community contribution. The contributors claim the databases work and we accept their pull request. So, I am unsure what the appropriate word to say here is.

Maybe "... from MaxMind are also supported by the community in this plugin" ?